appveyor.yml

```diff
@@ -113,7 +113,7 @@ install:
   - "activate test"
   # Install default dependencies
   # pip 10.0.1 seems buggy and breaks all our appveyor builds
-  - "conda install -q --yes \"pip<10\" numpy scipy matplotlib lxml sqlalchemy mock nose gdal decorator requests basemap jsonschema pyshp cryptography"
+  - "conda install -q --yes \"pip<10\" numpy scipy matplotlib lxml sqlalchemy mock nose gdal decorator requests basemap jsonschema pyshp cryptography \"proj4=4.9.3\""
```

Comment:
Could unpin proj4 here, but that would likely mean that plotting of maps wouldn't get testes on windows.